script.js

```diff
@@ -0,0 +1,53 @@
+// var choices = [1, 2, 3, 4, 5, 6];
+//maybe change "choices" to "Prompts" and the oucomes of prompts are the parameteres for moving on
+
+// var bluePill = false;
+// var redPill = true;
+
+
+
+var ready = prompt("Wake up, Neo. Are you ready?");
+
+ if (ready != null) {
+    confirm("Great. Follow the white rabbit");
+
+  }
+  else {
+  confirm("..but you have to be, Neo. You are the One. The Matrix has you. Follow the white rabbit. 'Knock, Knock")
+};
+
+  var meetMorpheus = prompt("After a bit of trouble, you meet Morpheus." + " " + "Morpheus says: 'Welcome, Neo. Do you know who I am?''");
+
+    if( meetMorpheus != null || "no" || "No") {
```

Comment:
i'm not going to comment on further ones, just know that if you ever do something like this : 

```js
someVar == 1 || 5
```

it will always return true. By virtue of the face that the value `5` is truthy.